src/Standards/Generic/Tests/ControlStructures/ControlStructuresBracketsUnitTest.inc

```diff
@@ -0,0 +1,170 @@
+<?php
+
+$a = true;
+
+// Warning
+if ($a)
+	echo true;
+
+// Fixable errors
+if ($a) {
+	echo true;
+}
+
+if ($a)
+
+{
+	echo true;
+}
+
+// Note the echo here will not be indented by this sniff but by PEAR's MultiLineConditionSniff
+if ($a)
+{echo true;
+}
+
+if ($a)
+     {
+	echo true;
+}
+
+function muh()
+{
+	if ($a)
+		{
+		echo true;
+	}
+
+	if ($a)
+{
+		echo true;
+	}
+
+	// If statement with spaces which aren't equal to a tab space (4 spaces)
+	if ($a)
+	  {
+		echo true;
+	}
+}
+
+// NewlineAfterOpenBrace
+if ($a)
+{
+
+	echo true;
+}
+
+while ($a < $b)
+{
+
+
+	echo 'muh';
+}
+
+// Closure indentation tests
+function foo()
+{
+	// Sort the array by value of subarray
+	usort(
+		$languages,
+		function($a, $b) use ($that)
+		{
+			$ordering = $that->getState('list.ordering');
+
+			if (strtolower($that->getState('list.direction')) === 'asc')
+		{
+				return StringHelper::strcmp($a->$ordering, $b->$ordering);
+			}
+			else
+		{
+				return StringHelper::strcmp($b->$ordering, $a->$ordering);
+			}
+		}
+	);
+}
+
+// Anonymous classes test
```

Comment:
There is a lot of real code in this test. I prefer to boil it down to only what is needed and then replace with fake code.

Reply:
@gsherwood I hope these changes are sufficient to obfuscate the "real code" and make it focused on just the structure being tested.